modules/combat-ffg.js

```diff
@@ -110,19 +110,18 @@ export class CombatFFG extends Combat {
               const baseFormulaType = container.querySelector('input[name="skill"]:checked').value;
 
               // Iterate over Combatants, performing an initiative roll for each
-              const [updates, messages] = ids.reduce(
-                (results, id, i) => {
+              const [updates, messages] = await ids.reduce(
+                async (results, id, i) => {
                   let [updates, messages] = results;
-
                   // Get Combatant data
                   const c = initiative.getCombatantByToken(
-                    initiative.data.combatants.map(combatant => combatant.data)
+                    initiative.combatants.map(combatant => combatant)
```

Comment:
This map isn't actually doing anything anymore so not necessary.

Not a blocker in our current fix-it-till-it-works maintenance world though. 😂